shared/chat/inbox/row/small-team/swipe-conv-actions/index.native.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react'
+import * as Kb from '../../../../../common-adapters/mobile.native'
+import * as Styles from '../../../../../styles'
+import {Props} from '.'
+
+let curSwipeRef: React.MutableRefObject<Kb.Swipeable> | null = null
+
+const onMute = (onMuteConversation: () => void, ref: React.MutableRefObject<Kb.Swipeable>) => {
+  ref.current && ref.current.close()
+  curSwipeRef = null
+  onMuteConversation()
+}
+
+const onHide = (onHideConversation: () => void, ref: React.MutableRefObject<Kb.Swipeable>) => {
```

Comment:
these two helpers can be one, just pass onHideConversation or onMuteConversation as param 1

Reply:
Sure